addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/ExtensionScriptsUI.java

```diff
@@ -457,12 +457,12 @@ public void displayScript(ScriptWrapper script, boolean allowFocus) {
                 this.getScriptsPanel().showInTree(node);
             }
 
-            if (!script.getEngine().isTextBased()) {
-                // Output message to explain about non test based scripts
-                this.getConsolePanel()
-                        .getCommandPanel()
-                        .setCommandScript(Constant.messages.getString("scripts.welcome.nontest"));
-            }
+            this.getConsolePanel()
+                    .getCommandPanel()
+                    .setScriptTooltip(
+                            script.getEngine().isTextBased()
+                                    ? null
+                                    : Constant.messages.getString("scripts.welcome.nontest"));
```

Comment:
test or text?

Or is the key actually wrong?

Reply:
The key is wrong...